### PR TITLE
chore!: move components to proper repositories

### PR DESCRIPTION
**Pull request by dreamwasp**
### Overview

Moves components to proper repositories, in accordance with the Gamut Audit notion

audit [here](https://www.notion.so/codecademy/gamut-labs-audit-d08a310c532b4ec880819ea1eb342e7d) for reference

### PR Checklist

- [x] Related to designs:
- [x] Related to JIRA ticket: web-1985
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/30762)  | [TBD](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/portal-app/pull/1968)  |
| BAM | [BAM Link](https://github.com/codecademy-engineering/business-account-management/pull/634) | [Another Env](http://www.google.fr/ 'Named link title')  |
| Author | [Author Link](https://github.com/codecademy-engineering/author/pull/2070) | [Another Env](http://www.google.fr/ 'Named link title')  |

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.127-alpha.a1afc5.0<br/>@codecademy/gamut-labs@31.12.2-alpha.a1afc5.0<br/>@codecademy/gamut@47.3.2-alpha.a1afc5.0<br/>@codecademy/monaco@0.6.45-alpha.a1afc5.0<br/>@codecademy/styleguide@57.4.1-alpha.a1afc5.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.127-alpha.da7f6e.0<br/>@codecademy/gamut-labs@31.12.2-alpha.da7f6e.0<br/>@codecademy/gamut@47.3.2-alpha.da7f6e.0<br/>@codecademy/monaco@0.6.45-alpha.da7f6e.0<br/>@codecademy/styleguide@57.4.1-alpha.da7f6e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62829ae251f8ab15599566bb--gamut-preview.netlify.app>https://62829ae251f8ab15599566bb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62829ae251f8ab15599566bb>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62829edab17a6000a0e1ea6b--gamut-preview.netlify.app>https://62829edab17a6000a0e1ea6b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62829edab17a6000a0e1ea6b>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.127-alpha.8fc3e7.0<br/>@codecademy/gamut-labs@31.12.2-alpha.8fc3e7.0<br/>@codecademy/gamut@47.3.2-alpha.8fc3e7.0<br/>@codecademy/monaco@0.6.45-alpha.8fc3e7.0<br/>@codecademy/styleguide@57.4.1-alpha.8fc3e7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6283f12b746cda00ac8b8428--gamut-preview.netlify.app>https://6283f12b746cda00ac8b8428--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6283f12b746cda00ac8b8428>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.127-alpha.e9256c.0<br/>@codecademy/gamut-labs@31.12.2-alpha.e9256c.0<br/>@codecademy/gamut@47.3.2-alpha.e9256c.0<br/>@codecademy/monaco@0.6.45-alpha.e9256c.0<br/>@codecademy/styleguide@57.4.1-alpha.e9256c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6283fb740eea0209f67768c3--gamut-preview.netlify.app>https://6283fb740eea0209f67768c3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6283fb740eea0209f67768c3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.128-alpha.d5559a.0<br/>@codecademy/gamut-labs@31.12.3-alpha.d5559a.0<br/>@codecademy/gamut@47.3.3-alpha.d5559a.0<br/>@codecademy/monaco@0.6.46-alpha.d5559a.0<br/>@codecademy/styleguide@57.4.2-alpha.d5559a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://628400b526b919128c0285d3--gamut-preview.netlify.app>https://628400b526b919128c0285d3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/628400b526b919128c0285d3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.128-alpha.861997.0<br/>@codecademy/gamut-labs@31.12.3-alpha.861997.0<br/>@codecademy/gamut@47.3.3-alpha.861997.0<br/>@codecademy/monaco@0.6.46-alpha.861997.0<br/>@codecademy/styleguide@57.4.2-alpha.861997.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://628549103ce55c293adcf0c6--gamut-preview.netlify.app>https://628549103ce55c293adcf0c6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/628549103ce55c293adcf0c6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.132-alpha.356d65.0<br/>@codecademy/gamut-labs@31.14.2-alpha.356d65.0<br/>@codecademy/gamut@47.3.4-alpha.356d65.0<br/>@codecademy/monaco@0.6.47-alpha.356d65.0<br/>@codecademy/styleguide@57.6.2-alpha.356d65.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://628d1b34ca833627e6d7b428--gamut-preview.netlify.app>https://628d1b34ca833627e6d7b428--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/628d1b34ca833627e6d7b428>Deploy Logs</a></p>

**Review by jakemhiller — APPROVED** [review]
Noice

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.132-alpha.663153.0<br/>@codecademy/gamut-labs@31.14.2-alpha.663153.0<br/>@codecademy/gamut@47.3.4-alpha.663153.0<br/>@codecademy/monaco@0.6.47-alpha.663153.0<br/>@codecademy/styleguide@57.6.2-alpha.663153.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://629638d0fa90f30084fb2be5--gamut-preview.netlify.app>https://629638d0fa90f30084fb2be5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/629638d0fa90f30084fb2be5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6296883119dc2a04f64fc58f--gamut-preview.netlify.app>https://6296883119dc2a04f64fc58f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6296883119dc2a04f64fc58f>Deploy Logs</a></p>

**Review by aresnik11 — APPROVED** [review]
🚀

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.132-alpha.b50710.0<br/>@codecademy/gamut-labs@31.14.2-alpha.b50710.0<br/>@codecademy/gamut@47.3.4-alpha.b50710.0<br/>@codecademy/monaco@0.6.47-alpha.b50710.0<br/>@codecademy/styleguide@57.6.2-alpha.b50710.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://629698853a82fa1215c23f95--gamut-preview.netlify.app>https://629698853a82fa1215c23f95--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/629698853a82fa1215c23f95>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.137-alpha.286509.0<br/>@codecademy/gamut-labs@31.15.5-alpha.286509.0<br/>@codecademy/gamut@47.4.1-alpha.286509.0<br/>@codecademy/monaco@0.6.49-alpha.286509.0<br/>@codecademy/styleguide@57.7.5-alpha.286509.0<br/></pre>

File: packages/gamut/src/Flyout/index.tsx [hunk 1]
```diff
@@ -1,8 +1,11 @@
-import { IconButton, Overlay, Text } from '@codecademy/gamut';
+// import { IconButton, Overlay, Text } from '@codecademy/gamut';
```
[the remark this answers is not in the document]
Comment (dreamwasp) [reply packages/gamut/src/Flyout/index.tsx:1]: yeup! 🙏

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.137-alpha.afbc6b.0<br/>@codecademy/gamut-labs@31.15.5-alpha.afbc6b.0<br/>@codecademy/gamut@47.4.1-alpha.afbc6b.0<br/>@codecademy/monaco@0.6.49-alpha.afbc6b.0<br/>@codecademy/styleguide@57.7.5-alpha.afbc6b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62a0aa79d0b1d81beedaa7af--gamut-preview.netlify.app>https://62a0aa79d0b1d81beedaa7af--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62a0aa79d0b1d81beedaa7af>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.137-alpha.300805.0<br/>@codecademy/gamut-labs@31.15.5-alpha.300805.0<br/>@codecademy/gamut@47.4.1-alpha.300805.0<br/>@codecademy/monaco@0.6.49-alpha.300805.0<br/>@codecademy/styleguide@57.7.5-alpha.300805.0<br/></pre>

**Pull request merged**